### PR TITLE
[pt2] add meta for `argsort.stable`, use `sort` samples in `OpInfo`

### PR DESCRIPTION
**Pull request by nkaretnikov**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #106025

### CODE REVIEW

**Comment by nkaretnikov** [comment]
From the list in https://github.com/pytorch/pytorch/issues/105105

File: test/functorch/test_vmap.py [hunk 1]
```diff
@@ -3739,6 +3739,7 @@ def test_vmap_exhaustive(self, device, dtype, op):
         # but found at least two devices, cuda:0 and cpu!
         xfail('ge', device_type='cuda'),
         xfail('_upsample_bilinear2d_aa'),
+        xfail('argsort'),  # aten::argsort.stable hit the vmap fallback which is currently disabled
```
Comment (nkaretnikov) [inline test/functorch/test_vmap.py:3742]: OpInfo now tests with the stable kwarg, so this got uncovered.

File: torch/testing/_internal/common_methods_invocations.py [hunk 2]
```diff
@@ -18269,7 +18265,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         "argsort",
         dtypes=all_types_and(torch.bool, torch.float16, torch.bfloat16),
         dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16),
-        sample_inputs_func=sample_inputs_argsort,
+        sample_inputs_func=sample_inputs_sort,
```
Comment (nkaretnikov) [inline torch/testing/_internal/common_methods_invocations.py:18268]: argsort is the second value returned by torch.sort(). Same API as sort, not sure why samples weren't reused before.

```
torch.argsort(input, dim=- 1, descending=False, stable=False) → Tensor
   torch.sort(input, dim=- 1, descending=False, stable=False, *, out=None)
```

Note: on the aten side, stable has no default:
```
func: argsort.stable(Tensor self, *, bool stable, int dim=-1, bool descending=False) -> Tensor
```

**Review by ezyang — APPROVED** [review]

**Review by zou3519 — APPROVED** [review]

**Pull request closed without merging**